core/client/fs/src/main/java/alluxio/client/file/cache/cuckoofilter/TagPosition.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.file.cache.cuckoofilter;
+
+/**
+ * This class represents a position of a tag.
+ */
+public class TagPosition {
+  public int mBucketIndex;
+  public int mSlotIndex;
+  public CuckooStatus mStatus;
```

Comment:
Do they have to be public?

Reply:
It will make my code more succinct if making these members public, since they are frequently accessed. I have made them private. After all, it doesn't fit in the java-style